examples/amp-story/invalid-link-tests.html

```diff
@@ -16,26 +16,29 @@
         background: white;
       }
       #link1 {
-        position: absolute;
-        top: 300px;
-        right: 320px;
+        position: relative;
+        top: 50%;
+        left: -10%;
         width: 30px;
```

Comment:
Looks great.
nit: You could consolidate shared properties like so:
```
#link1, #link2, #link3 {
  position: relative;
  width: 30px;
}
#link1 {
  top: 50%;
  left: -10%;
}
#link2 {
  top: 60%;
  left: 99%;
}
#link3 {
  top: 70%;
  left: 50%;
}
```
This way it's easier to see what is changing between similar elements.